CIScripts/Test/TestRunner.ps1

```diff
@@ -30,27 +29,36 @@ function Run-TestScenarios {
-        # TODO: Uncomment after JW-1129
-        # Test-SNAT -Session $Sessions[0] -SNATConfiguration $SNATConfiguration -TestConfiguration $TestConfiguration
-        Test-VRouterAgentIntegration -Session1 $Sessions[0] -Session2 $Sessions[1] `
-            -TestConfiguration $TestConfiguration -TestConfigurationUdp (Get-TestConfigurationUdp)
-        Test-ComputeControllerIntegration -Session $Sessions[0] -TestConfiguration $TestConfiguration
-        Test-MultipleSubnetsSupport -Session $Sessions[0] -TestConfiguration $TestConfiguration
-        Test-DockerDriverMultiTenancy -Session $Sessions[0] -TestConfiguration $TestConfiguration
-        Test-WindowsLinuxIntegration -Session $Sessions[0] -TestConfiguration (Get-TestConfigurationWindowsLinux)
-        Test-Pkt0PipeImplementation -Session $Sessions[0] -TestConfiguration $TestConfiguration
-
-        if($Env:RUN_DRIVER_TESTS -eq "1") {
-            Test-DockerDriver -Session $Sessions[0] -TestConfiguration $TestConfiguration
-        }
+        # Test-AgentService -Session $Sessions[0] -TestConfiguration $TestConfiguration
+        # Test-ExtensionLongLeak -Session $Sessions[0] -TestDurationHours $Env:LEAK_TEST_DURATION -TestConfiguration $TestConfiguration
+        # Test-MultiEnableDisableExtension -Session $Sessions[0] -EnableDisableCount $Env:MULTI_ENABLE_DISABLE_EXTENSION_COUNT -TestConfiguration $TestConfiguration
+        # Test-VTestScenarios -Session $Sessions[0] -TestConfiguration $TestConfiguration
+        # Test-TCPCommunication -Session $Sessions[0] -TestConfiguration $TestConfiguration
+        # Test-ICMPoMPLSoGRE -Session1 $Sessions[0] -Session2 $Sessions[1] -TestConfiguration $TestConfiguration
+        # Test-TCPoMPLSoGRE -Session1 $Sessions[0] -Session2 $Sessions[1] -TestConfiguration $TestConfiguration
+        # # TODO: Uncomment after JW-1129
+        # # Test-SNAT -Session $Sessions[0] -SNATConfiguration $SNATConfiguration -TestConfiguration $TestConfiguration
+        # Test-VRouterAgentIntegration -Session1 $Sessions[0] -Session2 $Sessions[1] `
+        #     -TestConfiguration $TestConfiguration -TestConfigurationUdp (Get-TestConfigurationUdp)
+        # Test-ComputeControllerIntegration -Session $Sessions[0] -TestConfiguration $TestConfiguration
+        # Test-MultipleSubnetsSupport -Session $Sessions[0] -TestConfiguration $TestConfiguration
+        # Test-DockerDriverMultiTenancy -Session $Sessions[0] -TestConfiguration $TestConfiguration
+        #Test-WindowsLinuxIntegration -Session $Sessions[0] -TestConfiguration (Get-TestConfigurationWindowsLinux)
+        # Test-Pkt0PipeImplementation -Session $Sessions[0] -TestConfiguration $TestConfiguration
+        
+        # if($Env:RUN_DRIVER_TESTS -eq "1") {
+        #     Test-DockerDriver -Session $Sessions[0] -TestConfiguration $TestConfiguration
+        # }
     })
+    if($Env:RUN_PESTER_TESTS -eq "1") {
```

Comment:
Please future-proof the code for strict mode by adding `(Test-Path Env:RUN_PESTER_TESTS) -and `

Please add space after `if` (here and in all the other places)